OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
No, not to target structures in general, it's used to give more weight (Attractiveness) for targeting construction yards sepcifically i.e. base building so it can be used like this so the bot would disable the bot opponents capability to build new buildings to recover after being hit by the nukes for instance

			OrderName: NukePowerInfoOrder
			MinimumAttractiveness: 3000
			Consideration@1:
				Against: Enemy
				Types: BaseBuilding
				Attractiveness: 1000
				TargetMetric: Value
				CheckRadius: 3c0